.github/workflows/ci.yml

```diff
@@ -118,21 +118,21 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
         db-backend: ["postgresql"]
-        nautobot-version: ["1.4.0b1"]  # Change to latest or stable
+        nautobot-version: ["1.4.0", "latest"]
```

Comment:
We don't need a full matrix, just latest here, same as fw https://github.com/nautobot/nautobot-plugin-firewall-models/blob/develop/.github/workflows/ci.yml#L116

Reply:
```suggestion
        nautobot-version: ["latest"]
```